src/Core/Services/Implementations/PolicyService.cs

```diff
@@ -44,6 +44,24 @@ public class PolicyService : IPolicyService
             {
                 throw new BadRequestException("This organization cannot use policies.");
             }
+            
+            // Handle dependent policy checks
+            switch(policy.Type)
+            {
+               case PolicyType.RequireSso:
+                   if (policy.Enabled)
+                   {
+                       var singleOrg = await _policyRepository.GetByOrganizationIdTypeAsync(org.Id, PolicyType.OnlyOrg);
```

Comment:
If you get latest from `master`, the `PolicyType` should be `SingleOrg`. This would break if merged I would imagine.

Reply:
Very true- merging in master and then updating the change!